circle.yml

```diff
@@ -110,3 +110,9 @@ experimental:
     branches:
       only:
         - staging
+deployment:
+  storybook:
+    branch: staging
+    commands:
+      - 'echo "sync_assets: false" >> locals.yml'
+      - 'cd apps && npm run storybook:deploy'
```

Comment:
Yeah, I'd temporarily add this branch to the config and try the deployment from here a few times.